src/reporter.ts

```diff
@@ -240,42 +243,42 @@ class RPReporter implements Reporter {
     }
   }
 
-  onTestBegin(test: TestResp): void {
-    const suitesOrder: string[] = [];
-    const createSuitesOrder = (suite: any) => {
-      if (!suite?._isDescribe) {
-        return;
-      }
-      suitesOrder.push(suite.title);
-      createSuitesOrder(suite.parent);
-    };
-    createSuitesOrder(test.parent);
+  createSuitesOrder(suite: any): void {
+    if (!suite?._isDescribe) {
+      return;
+    }
+    this.suitesOrder.push(suite.title);
```

Comment:
Do not hold this object `suitesOrde` on the class level, it should be some kind of local variable as you done it previously.